README.md

```diff
@@ -1,4 +1,89 @@
-# WAYF Cloud Javascript Client
+# WAYF Cloud node.js Client
```

Comment:
Yes, this is for server applications. I can't think of a use case that the WAYF Cloud API would be used in a browser environment. 
I like your suggestion about renaming the repository.